postman/Get Clicks.postman_collection.json

```diff
@@ -0,0 +1,596 @@
+							"listen": "test",
+							"script": {
+								"id": "c8c9f482-117c-4cc9-87c7-04dbb1e65bfb",
+								"exec": [
+									"pm.test(\"Successfully got a broadcast\", function(){",
+									"    var jsonData = pm.response.json()",
+									"    pm.globals.set(\"broadcastId\", jsonData.broadcastId)",
+									"})",
+									""
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"url": {
+							"raw": "{{broadcast_self_link}}",
+							"host": [
+								"{{broadcast_self_link}}"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Get Campaign",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "5ee2b5ae-dea0-457f-8ac3-55015cb78cec",
+								"exec": [
+									"pm.test(\"Successfully got a campaign\", function(){",
+									"    var jsonData = pm.response.json();",
+									"    pm.globals.set(\"messages_collection_link\", jsonData.messages_collection_link)",
+									"})"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"url": {
+							"raw": "{{base_url}}/accounts/{{accountId}}/lists/{{listId}}/campaigns/b{{broadcastId}}",
+							"host": [
+								"{{base_url}}"
+							],
+							"path": [
+								"accounts",
+								"{{accountId}}",
+								"lists",
+								"{{listId}}",
+								"campaigns",
+								"b{{broadcastId}}"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Get Messages in Broadcast",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "0826d611-cf14-4a5c-a835-ac43669d9fca",
+								"exec": [
+									"pm.test(\"Successfully got the messages in the broadcast\", function(){",
+									"    ",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"url": {
+							"raw": "{{messages_collection_link}}",
+							"host": [
+								"{{messages_collection_link}}"
+							]
+						}
+					},
+					"response": []
+				}
+			]
+		},
+		{
+			"name": "Get Followup Email Clicks",
+			"item": [
+				{
+					"name": "Get Number of Followups on List",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "1bac9b8a-de19-4264-9c19-bdc1ea154a5c",
+								"exec": [
+									"pm.test(\"Successfully got number of followups in list\", function() {",
+									"    pm.expect(parseInt(responseBody)).to.be.a('number');",
+									"    var followup_entry = Math.floor((Math.random() * parseInt(responseBody) ))",
+									"    pm.globals.set(\"followup_entry\",followup_entry )",
```

Comment:
You shouldn't use this number to pick a random followup from the first page. Only the first 100 entries are included in that page, while total_size shows the total number of followups on the list.

Reply:
Updated